scripts/local_test/README.md

```diff
@@ -0,0 +1,70 @@
+# Operating requirement
+
+- Run the scripts with bash, because we need to load env variables from cron.yml
+
+- Run the third script in common user, the others in root
+
+- Run 4.1 after `cargo build --release`
+
+- If there's problem after 4.2, just check your proxy setting and switch proxy node.
+
+- Make sure the virtual machine has enough resouces, 8g mem, 4 cores, 200g disk.
+
+# Practical steps
+
+1. Create a vm for test. Install some basic software: ssh, vim, cmake, protobuf, rust, curl, take snapshot for the vm as Snapshot1
```

Comment:
Can we write a script for setting up the development environment based on the information in `ci.yml`?

Reply:
That's not necessary because different systems may have different install way. And a snapshot will be token after them installed.